src/class_objectProcessor.py

```diff
@@ -1088,15 +1106,16 @@ def ackDataHasAValidHeader(self, ackData):
         return True
 
     def addMailingListNameToSubject(self, subject, mailingListName):
+        """"""
         subject = subject.strip()
         if subject[:3] == 'Re:' or subject[:3] == 'RE:':
             subject = subject[3:].strip()
         if '[' + mailingListName + ']' in subject:
             return subject
-        else:
-            return '[' + mailingListName + '] ' + subject
+        return '[' + mailingListName + '] ' + subject
 
     def decodeType2Message(self, message):
```

Comment:
Deprecated, remove.

Reply:
Removing the line `return '[' + mailingListName + '] ' + subject`, formerly the else clause. This means `None` will be returned at the end of the method. Is this what you meant? Or did you want `subject` returning in any case?

Reply:
@coffeedogs `decodeType2Message` is deprecated

Reply:
Right, must have already removed `decodeType2Message`. Github's line indication is very off-putting sometimes. Reverting the `return` statement there, though removing the `else:` as lint.

Reply:
It probably happens if you replace your old commit with a modified one.